LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright 2024 Rust Basel
+Copyright 2023 Thomas Weitzel
```

Comment:
Copyright 2025 Rust Basel

Reply:
Fixed in 8835975daec0710a4b4b16d5cabed44768fbadd0